drizzlepac/hlautils/astrometric_utils.py

```diff
@@ -70,57 +68,36 @@
 
 MODULE_PATH = os.path.dirname(inspect.getfile(inspect.currentframe()))
 VEGASPEC = os.path.join(os.path.dirname(MODULE_PATH),
-                        'data','alpha_lyr_stis_008.fits')
+                        'data', 'alpha_lyr_stis_008.fits')
 
 __all__ = ['create_astrometric_catalog', 'compute_radius', 'find_gsc_offset',
-            'extract_sources', 'find_hist2d_offset', 'generate_source_catalog',
-            'classify_sources', 'countExtn']
+           'extract_sources', 'find_hist2d_offset', 'generate_source_catalog',
+           'classify_sources', 'countExtn']
 
-"""
-Utility functions.
 
-Many were originally released in stsci.tools.fileutil.
-"""
 def buildRotMatrix(theta):
```

Comment:
If `countExtn` was removed in favor of using it directly from stsci.tools, the same could be done with this function as well.